### PR TITLE
Auto upload crash if filepath is null

### PR DESCRIPTION
**Pull request by tobiasKaminsky**
Ref: https://github.com/nextcloud/android/issues/1100#issuecomment-315737685

Also changed the + to white, so it fits to the number.

### CODE REVIEW

File: src/main/res/layout/grid_sync_item.xml [hunk 1]
```diff
@@ -58,7 +58,7 @@
                 android:layout_width="wrap_content"
                 android:layout_height="wrap_content"
                 android:text="+"
-                android:textColor="#bcbcbc"
+                android:textColor="#ffffff"
```
Comment (AndyScherzinger) [inline src/main/res/layout/grid_sync_item.xml]: why did you turn it into white? The slight grey was actually on purpose :D

**Comment by AndyScherzinger** [comment]
👍 fine by me, as stated in the comment I explicitly made the + slightly grey-ish to focus on the number (like facebook does it)... But I am fine either way.

File: src/main/res/layout/grid_sync_item.xml (hunk 1 above)
Comment (tobiasKaminsky) [inline src/main/res/layout/grid_sync_item.xml]: umm...ok...You are the ui guru ;-)
Lukas and me were just wondering and grey was a little bit hard to read.
Your decision :+1: or :-1: (like Ceasar)

Comment (AndyScherzinger) [inline src/main/res/layout/grid_sync_item.xml]: 👍 let's keep #ffffff then

**Comment by AndyScherzinger** [comment]
do we now merge with a single +1 for the time being?

**Comment by tobiasKaminsky** [comment]
This is a general "problem" if one of us is on vacation.
On short vacations (up to 1 week) we can just wait, I think. Mario will be available next week again, so I guess we can wait.
On longer vacations, at least I, will always have a short look into github to reply/review/postpone.

-> wait if it is not urgent, otherwise merge (should be an exception)

**Comment by AndyScherzinger** [comment]
I have to say I completely disagree. "wait for a week" means "postpone a release by a week" since in my opinion a roadblock for a week means it'll take a week longer than planned. No integration of changes will take place so you will have to spend that effort "a week later" or descope stuff none of which I think should be the standard. 

The basic issue is that the 4-eyes principle doesn't work all the time for a 3 people core team but teams >=4 people...

Solution: let's find another dev who contributes regularly (to be found) and is willing to do reviews (to be found)... 😉

**Comment by tobiasKaminsky** [comment]
> Solution: let's find another dev who contributes regularly (to be found) and is willing to do reviews (to be found)

Yes, this is the best solution.
But in the meantime, at lest in my opinion, it is quality over speed.

**Comment by AndyScherzinger** [comment]
I'm fine with that and would say let's discuss this at the conference if we can't find another, yet better approach :)

**Comment by mario** [comment]
Conflicts, otherwise ok.

**Comment by AndyScherzinger** [comment]
@mario I resolved the conflicts (Lint), so merging... 🎉

**Pull request merged**